src/style.css

```diff
@@ -1 +1,65 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0 0 50px;
+}
+
+.nav {
+  display: inline-block;
+  margin: auto 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  background-color: #fff;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin: 0 10px;
+}
+
+.nav__link {
+  display: inline-block;
+  color: #000;
+  text-decoration: none;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
```

Comment:
move font-family: "Roboto", sans-serif; to html tag

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3081/files#r1184154997
not fixed

move to `html` tag selector